.github/workflows/nightly-noinstall.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           apiVersion: kind.x-k8s.io/v1alpha4
           nodes:
           - role: control-plane
+          - role: worker
```

Comment:
Is there a reason that you're setting the ingress configuration and port mappings in the worker node instead of the the control-plane node?

Reply:
it shouldn't matter, we just need to ensure that the ingress e2e test runs on the node that this configuration is set for only. I have left the configuration on the control-plane node itself

Reply:
we want the nginx controller pod to be scheduled on a specific node, based on the node selector match `ingress-reday=true` . Since the control-plane node has a master label, the pod nginx pod won't be scheduled on this even if the `inress-ready=true` label selector matches, hence I moved it to a worker.